js/src/weekly_report.js

```diff
@@ -0,0 +1,60 @@
+// # Generate a weekly report of completed issues per repo
+const fetch = require('node-fetch')
+const {
+  compose,
+  zipObj,
+  tap,
+  mapAwait,
+  tapLog,
+  prop,
+  gte,
+  map,
+  filter,
+  branch,
+  entries,
+} = require('./utils')
+
+// check for arg
+const date = process.argv?.[2]
+if (!date) {
+  throw new Error(
+    'Please call this script with a date in the YYYY-MM-DD format.'
+  )
+}
+
+// data
+const repos = [
+  'openverse',
+  'openverse-frontend',
+  'openverse-api',
+  'openverse-catalog',
+]
+
+// internals
+const API_ROOT = `https://api.github.com`
+const makeIssueUrl = owner => repo =>
+  `${API_ROOT}/repos/${owner}/${repo}/issues?state=closed&since=${date}&per_page=10000`
+const makeWpIssueUrl = makeIssueUrl('wordpress')
+// this key only has public repo access, i don't care about showing it
+const authFetch = url =>
+  fetch(url, {
+    headers: {
+      Authorization:
+        'Basic emFja2tyaWRhOmdocF9JVHdJM2ZGUkVqSjZkOWRBUFFBOE5pWktwNlU0dXkxRTVFak8=',
```

Comment:
A key for is not needed for reading public resources.

Reply:
Is useful locally however, for the purpose of rate limiting. Maybe I'll make it an optional script input.